turbinia/workers/volatility.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Task for executing volatility."""
+
+from __future__ import unicode_literals
+
+import os
+
+from turbinia.evidence import VolatilityReport
+from turbinia.workers import TurbiniaTask
+
+
+class VolatilityTask(TurbiniaTask):
+  """Task to execute volatility."""
+
+  def __init__(self, module='test'):
+    super(VolatilityTask, self).__init__()
+    self.module = module
+
+  def run(self, evidence, result):
+    """Task to execute volatility."""
+
+    # Create the new Evidence object that will be generated by this Task.
+    output_evidence = VolatilityReport()
+
+    # Create a path that we can write the new file to.
+    output_file_path = os.path.join(
+        self.output_dir, '{0:s}.txt'.format(self.id))
+
+    output_evidence.local_path = output_file_path
```

Comment:
I think the reporting is new since you started this PR, but you could also read the contents of this file and then put it in `result.report_data`, and that way the output could show up in the reports.

Reply:
BTW, there's some more at https://github.com/google/turbinia/blob/master/docs/developing-new-tasks.md#reporting on how the reporting works.